website/src/pages/dictionary/Schema/index.js

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+import Table from '@icgc-argo/uikit/Table';
+import TagButton, { TAG_TYPES } from './TagButton';
+import styles from './styles.module.css';
+import Tag from '@icgc-argo/uikit/Tag';
+import CodeList from './CodeList';
+import Regex from './Regex';
+
+const formatFieldType = value => {
+  switch (value) {
+    case 'string':
+      return 'TEXT';
+    default:
+      return value.toUpperCase();
+  }
+};
+
+const FieldDescription = ({ name, description }) => (
+  <div className={styles.fieldDescription}>
+    <div className={styles.name}>{name}</div>
+    <div>{description}</div>
+  </div>
+);
+
+const FieldsTag = ({ fieldCount }) => (
+  <Tag className={styles.fieldsTag}>{`${fieldCount} Field${fieldCount > 1 ? 's' : ''}`}</Tag>
+);
+
+const Schema = ({ schema, key }) => {
+  /**
+   * need to pass in state for Cell rendering
+   * react-table rerenders everything, change shape of codelist to pass in state
+   */
+  const expandingFields = schema.fields.reduce((acc, val) => {
+    acc[val.name] = false;
+    return acc;
+  }, {});
+
+  const [expandedCodeLists, setExpandedCodeLists] = useState(expandingFields);
```

Comment:
should we add a side effect to reset `expandedCodeLists` to the recomputed `expandingFields` when `schema` changes? 

on that note, `initialExpandingFields` might be a better name because it's only being used as the initial state:)